starters/graphql-serverless-contentful/src/schema/post/post.resolver.test.ts

```diff
@@ -0,0 +1,204 @@
+import gql from 'graphql-tag';
+import { apolloServer } from '../../handlers/graphql';
+import { GraphQLResponse } from '@apollo/server/src/externalTypes';
+import assert from 'assert';
+
+const MOCK_CONTENT = [
+	'This is the test post content',
+	'This is the updated test post content',
+];
+
+jest.mock('../../utils/contentful', () => {
```

Comment:
This comment is a nice to have. Feel free to leave things as is.
I just remembered how we could turn this around, so we don't need the double declaration anymore. Here's a small example. If that doesn't clarify things, just ping me.

```JavaScript
import { myService } from '../myService';

jest.mock('../myService', () => ({ getClient: jest.fn() }));
const MOCKED_COMMENTS = ['This is a cool comment', 'This is an angry comment'];
myService.getClient.mockReturnValue({ getEntries: () => MOCKED_COMMENTS });
```
```

Reply:
cc @iansamz, you might like this as well.